translations/pl.json

```diff
@@ -2084,7 +2084,9 @@
                         "header": "Konfiguracja widoku",
                         "header_name": "Konfiguracja widoku {name}",
                         "move_left": "Przesuń widok w lewo",
-                        "move_right": "Przesuń widok w prawo"
+                        "move_right": "Przesuń widok w prawo",
+                        "tab_settings": "Ustawienia",
```

Comment:
Don't change the Polish translations here, but wait till they are available in Lokalise.

Reply:
Removed. I've added this to test if it works. It's my first PR in this repo